src/styles/blocks/item.scss

```diff
@@ -0,0 +1,89 @@
+.item {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  font-family: ' Roboto ', sans-serif;
```

Comment:
```suggestion
  font-family: 'Roboto', sans-serif;
```

Reply:
![image](https://github.com/user-attachments/assets/cf5d3c92-d5b1-4f9f-a510-ce56d2d132e2)

Reply:
Then you could try without quotes at all ar wit double quotes: `"Roboto"`